From ae4a9a7bef197ea21291b85737e6ac8a90829d0f Mon Sep 17 00:00:00 2001
From: Wei-Luan Wang <weiluanwang@google.com>
Date: Mon, 22 Dec 2025 09:35:37 +0000
Subject: [PATCH] Remove const in RunLoop trace event

As a workaround to fix build error in
crrev.com/c/7292081/comments/62e20375_aa9c3f7f?tab=comments

Change-Id: I152ceaf92dd178d73e7dfdfa789edd98a332f065
---
 base/run_loop.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/run_loop.cc b/base/run_loop.cc
index b387350220..037fd8a4d7 100644
--- a/base/run_loop.cc
+++ b/base/run_loop.cc
@@ -62,7 +62,7 @@ RunLoop::Delegate::~Delegate() {
 }
 
 bool RunLoop::Delegate::ShouldQuitWhenIdle() {
-  const auto* top_loop = active_run_loops_.top().get();
+  auto* top_loop = active_run_loops_.top().get();
   if (top_loop->quit_when_idle_) {
     TRACE_EVENT("toplevel.flow", "RunLoop_ExitedOnIdle",
                 perfetto::TerminatingFlow::FromPointer(top_loop));
-- 
2.52.0.322.g1dd061c0dc-goog

